<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CCardChangeMenu</class>
 <widget class="QDialog" name="CCardChangeMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>588</width>
    <height>593</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="headline">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>30</y>
     <width>391</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string>Karten Eintauschen</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>521</width>
     <height>171</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="cardsLayout"/>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>420</y>
     <width>361</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="PBLayout">
    <item>
     <widget class="QPushButton" name="changePB">
      <property name="text">
       <string>Tauschen</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="quitPB">
      <property name="text">
       <string>Schließen</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="informationLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>350</y>
     <width>421</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Anzahl der Einheiten die Sie beim Tausch erhalten würden:</string>
   </property>
  </widget>
  <widget class="QLabel" name="unitsLabel">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>350</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
